src/components/Recorder.js

```diff
@@ -107,6 +109,17 @@ export default class Recorder extends H5P.EventDispatcher {
     return loadAudioUrl;
   }
 
+  _uploadAudioToServer(blob) {
+    const reader = new FileReader();
+    reader.readAsDataURL(blob);
+    reader.onloadend = function() {
+        const base64data = reader.result;
+        dispatchEvent(new CustomEvent('tapestry-h5p-audio-recorder', { 
```

Comment:
I think it would be useful to also send out the id of the H5P content if it's possible

Reply:
I don't think the ID is generated in this code base. Do we really want to have the ID? And what would we need it for?

Reply:
I investigated this. The H5P ID could be retrieved and is now sent to the backend on the Vue side.